lib/src/policy/granter.ts

```diff
@@ -0,0 +1,180 @@
+import { Attribute, AttributeRuleType, KeyAccessServer, Value } from './attributes.js';
+
+export type KeyAccessTemplate = {
```

Comment:
nit: maybe rename to keysplitstep to keep in line with other sdks https://github.com/opentdf/platform/blob/d9a07f84ab5686fe13af82435af3201042dd7228/sdk/granter.go#L31

Reply:
Done